contracts/smart-contract-wallet/MaliciousAccount.sol

```diff
@@ -159,18 +162,12 @@ contract MaliciousAccount is
 
 
     // init
-    // Initialize / Setup
-    // Used to setup
-    // i. owner ii. entry point address iii. handler
-    function init(address _owner, address _entryPointAddress, address _handler) public override initializer { 
+    function init(address _owner, address _handler) public override initializer { 
         require(owner == address(0), "Already initialized");
-        require(address(_entryPoint) == address(0), "Already initialized");
         require(_owner != address(0),"Invalid owner");
-        require(_entryPointAddress != address(0), "Invalid Entrypoint");
-        require(_handler != address(0), "Invalid Entrypoint"); // not good :( 
+        require(_handler != address(0), "Invalid Fallback Handler");
         owner = _owner;
-        _entryPoint =  IEntryPoint(payable(_entryPointAddress));
-        if (_handler != address(0)) internalSetFallbackHandler(_handler);
+        internalSetFallbackHandler(_handler);
```

Comment:
I would suggest renaming it to _setFallbackHandler

Reply:
you mean the function name internalSetFallbackHandler or the variable name _handler ?

Reply:
I mean function name internalSetFallbackHandler => _setFallbackHandler

Reply:
ok done